.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+### Feature Requests ###
+Please read https://github.com/jquery/jquery/wiki/Adding-new-features ; most features should be implemented as plugins outside of jQuery.
+
+### But Reports ###
+Note that we only can fix bugs in the latest (1.x, 2.x, 3.x) versions of jQuery. Briefly describe the issue you've encountered. What you expect to happen? What acually happens? Which browsers are affected?
+
+
+Provide a minimal test case to demonstrate, using http://jsbin.com or http://jsfiddle.net ; if this isn't possible, start by asking for help on http://forum.jquery.com or http://stackoverflow.com to narrow down the problem and create a simple test case.
```

Comment:
Maybe that link is outdated though, and not relevant for jQuery. Does anybody knows a better one?